src/DashboardService/Controllers/PrioritiesController.cs

```diff
@@ -16,40 +18,42 @@ public class PrioritiesController : ControllerBase
 {
   [HttpPost]
   public async Task<OperationResultResponse<Guid?>> CreateAsync(
-    [FromServices] CreatePriorityCommand command,
+    [FromServices] ICreatePriorityCommand command,
     [FromBody] CreatePriorityRequest request)
   {
     return await command.ExecuteAsync(request);
   }
   
   [HttpGet]
   public async Task<FindResultResponse<PriorityInfo>> GetAsync(
-    [FromServices] GetPrioritiesCommand command,
-    [FromQuery] GetPrioritiesFilter filter)
+    [FromServices] IGetPrioritiesCommand command,
+    [FromQuery] GetPrioritiesFilter filter,
+    CancellationToken ct)
   {
-    return await command.ExecuteAsync(filter);
+    return await command.ExecuteAsync(filter, ct);
   }
   
   [HttpGet("{id}")]
   public async Task<OperationResultResponse<PriorityInfo>> GetAsync(
-    [FromServices] GetPriorityCommand command,
-    [FromRoute] Guid id)
+    [FromServices] IGetPriorityCommand command,
+    [FromRoute] Guid id,
+    CancellationToken ct)
   {
-    return await command.ExecuteAsync(id);
+    return await command.ExecuteAsync(id, ct);
   }
 
   [HttpPatch("{id}")]
   public async Task<OperationResultResponse<bool>> EditAsync(
-    [FromServices] EditPriorityCommand command,
+    [FromServices] IEditPriorityCommand command,
     [FromRoute] Guid id, 
-    [FromBody] JsonPatchDocument<PatchPriorityRequest> request)
+    [FromBody] JsonPatchDocument<EditPriorityRequest> request)
```

Comment:
In some endpoints you use CancellationToken ct, in some not. Why?

Reply:
added to all endpoints